server/rest/src/main/java/org/infinispan/rest/resources/ServerResource.java

```diff
@@ -122,6 +125,26 @@ public Invocations getInvocations() {
             .build());
    }
 
+   private CompletionStage<RestResponse> report(RestRequest restRequest) {
+      ServerManagement server = invocationHelper.getServer();
+      NettyRestResponse.Builder responseBuilder = new NettyRestResponse.Builder();
+      return server.getServerReport().handle((path, t) -> {
+         if (t != null) {
+            return responseBuilder.status(HttpResponseStatus.INTERNAL_SERVER_ERROR).build();
+         } else {
+            return responseBuilder
+                  .contentType(MediaType.fromString("application/gzip"))
+                  .header("Content-Disposition",
```

Comment:
What is this header for?

Reply:
It forces the browser to download and sets the name of the file.